content/blog/eclipse-temurin-slsa2/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: SLSA level 2 compliance for Eclipse Temurin
+date: "2022-11-11T12:00:00+00:00"
+author: sxa
+description: Eclipse Temurin by Adoptium is compliant with level 2 of the SLSA secure development framework.
+tags:
+  - Temurin
+  - Security
+---
+
+# Introduction to SLSA
+
+[SLSA](https://slsa.dev) is a framework with individual levels to indicate
+increasing levels of security which software products can work towards. 
```

Comment:
IMHO the SLSA website is a bit confused about sometimes addressing the supply chain vs. defining a level of general software development/consumption security. It certainly complements our wider security efforts. It may be better to paraphrase the purpose as described on their website to avoid a misunderstanding.

```suggestion
[SLSA](https://slsa.dev) is a framework with individual levels that software producers can work towards to make their software more secure, and consumers can make decisions based on the software package’s security posture. 
```